mac/build.sh

```diff
@@ -252,8 +252,10 @@ if $CLEAN ; then
     do_clean
 fi
 
-carthage bootstrap
-
+if [ "$TEST_ACTION" == "test" ]; then
+	carthage bootstrap		
```

Comment:
The test target still uses Carthage to get OCMocks. It looks like we should be able to change this to use cocoapods as well. (IIRC, I originally didn't use cocoapods for that framework because the build machines already had Carthage and the cocoapods installation looked scary.)

Reply:
k. Just checking if the test target still builds. It looks like you removed the Carthage files?
Also, see why the PR build failed

Reply:
The Carthage file used for the test target is Carthage.private.

Reply:
Fixed the build problem. Test build is happy.